### DISCUSSION TITLE
Demo supported platforms: iOS?

### DISCUSSION THREAD

**Discussion by H123334444** [j20001970/GDMP-demo#13] [Q&A]
Is there a demo available for exporting an ipa on macOS to facilitate testing on iOS devices, or does GDMP run on macOS without the need for precompiled libraries? Thank you so much for answering.

**Comment by j20001970** [comment]
There is currently no precompiled libraries for macOS/iOS unfortunately, to develop on these platforms, a computer running macOS, Apple ID and stuff is required and I have none of them. Basically I try to maintain GDMP for these Apple platforms without actual tools for them.

That being said, you can still try to compiling GDMP from source for macOS/iOS if you have required setup:
- For building macOS library(.dylib), after completing [common setup for GDMP](https://github.com/j20001970/GDMP#getting-started), follow [official MediaPipe documentation](https://developers.google.com/mediapipe/framework/getting_started/install#installing_on_macos) to install OpenCV, then run `build.py` with the same usage as building for Linux.
- For building iOS library, there is already a [building instruction for iOS,](https://github.com/j20001970/GDMP#building-for-ios) however these steps are considered deprecated and may not work anymore.

If you would like to contribute to macOS/iOS related improvement of GDMP it will be appreciated.
